src/modules/commentModal.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-unresolved
+import TVShowApp from '../App';
+
+const commentModal = async (showId, shows) => {
+  const show = shows.find((t) => t.id === parseInt(showId, 10));
+
+  const commentModalView = document.getElementById('comment-modal');
+  commentModalView.style.display = 'block';
+
+  const imgEl = commentModalView.querySelector('#show-image');
+  const showTitleEl = commentModalView.querySelector('.movie-info #title');
+  const showSummaryEl = commentModalView.querySelector('.movie-info #movie-summary');
+  const commentCounter = commentModalView.querySelector('.comments #comment-counter');
+  const commentList = commentModalView.querySelector('.comments #comment-list');
+  const closeEl = commentModalView.querySelector('.close');
+  const showIdEl = commentModalView.querySelector('form #show_id');
+  showIdEl.value = showId;
+
+  closeEl.addEventListener('click', () => { commentModalView.style.display = 'none'; });
+
+  imgEl.src = show.image;
+  showTitleEl.textContent = show.title;
+  showSummaryEl.innerHTML = show.summary;
+
+  const response = await TVShowApp.getComments(showId);
+  commentCounter.textContent = `(${typeof response.length === 'undefined' ? 0 : response.length})`;
+
+  // eslint-disable-next-line space-before-blocks
```

Comment:
- [ ] Kindly fix all programming errors flagged by Linters. Do NOT disable it. Disabling means you are changing their rules. This is not accepted.